config/src/main/java/com/typesafe/config/impl/SimpleConfig.java

```diff
@@ -850,10 +848,21 @@ static MemoryUnit parseUnit(String unit) {
      */
     public static long parseBytes(String input, ConfigOrigin originForException,
             String pathForException) {
+        BigInteger result = parseBytesAsBigInteger(input, originForException, pathForException);
```

Comment:
hmm - is the one returning a `long` still called? this `impl` package is private and we can delete this method if unused.

Reply:
you're right, I've weeded it out. thank you.